content/en/docs/concepts/services-networking/service.md

```diff
@@ -431,14 +431,12 @@ this case, you can create what are termed “headless” Services, by explicitly
 specifying `"None"` for the cluster IP (`.spec.clusterIP`).
 
 You can use a headless Service to interface with other service discovery mechanisms,
-without being tied to Kubernetes' implementation. For example, you could implement
-a custom {{< glossary_tooltip term_id="operator-pattern" text="Operator" >}} upon
-this API.
+without being tied to Kubernetes' implementation.
 
```

Comment:
hello @sftim .
I reread the section. A few nits and suggestions.
- hyphenate load-balancing or not?
- What does this phrase mean, `without being tied to Kubernetes' implementation.`? Implementation of what?
- line 454, Is it reasonable to double quote, "A record"? Also, link to https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-records
- Should `Services` be bold, if so, be consistent
- bold `kube-proxy`?
- I think line 436 could be broken down and made into a list:
  ### Limitations of Headless Services:
   - a cluster IP is not allocated
   - `kube-proxy` is not in use
   - Kubernetes does not perform load balancing 
- line 443, should it be `Endpoints Controller` instead of endpoints controller?

Reply:
Whilst that's legitimate feedback, I'd like to get this 3-line change done now and have another PR take care of bigger fixes.

Reply:
Okay -- your intention is to just clean up the reference to Operators.